ui/components/component-library/popover-header/popover-header.test.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable jest/require-top-level-describe */
+import { render, fireEvent } from '@testing-library/react';
+import React from 'react';
+import { PopoverHeader } from './popover-header';
+
+describe('Popover', () => {
```

Comment:
nit: Change to PopoverHeader
```suggestion
describe('PopoverHeader', () => {
```